src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1326,7 +1330,9 @@ public void onPrefsLoaded(SharedPreferences preferences) {
     /* package */ UpdatesFromMixpanel constructUpdatesFromMixpanel(final Context context, final String token) {
         if (Build.VERSION.SDK_INT < MPConfig.UI_FEATURES_MIN_API) {
             Log.i(LOGTAG, "Web Configuration, A/B Testing, and Dynamic Tweaks are not supported on this Android OS Version");
-            return new UnsupportedUpdatesFromMixpanel(sSharedTweaks);
+            return new NoOpUpdatesFromMixpanel(sSharedTweaks);
+        } else if (mConfig.getDisableViewCrawler()) {
```

Comment:
2 separate if clauses make it clear that we are handling 2 separate cases here. 

1. API of the device it too old
1. turned off by configuration